source/language/classical.rst

```diff
@@ -110,6 +110,43 @@ multiplication, and division and the corresponding assignment operators.
    angle[10] c;
    c = angle(a + b); // cast to angle[10]
 
+
+Evaluation order
+~~~~~~~~~~~~~~~~
+
+OpenQASM evaluates expressions from left to right.
+
+   .. table:: [operator-precedence] operator precedence in OpenQASM.
+
+      +-------------------------------+---------------------------------------+
+      | Operator                      | Descriptions                          |
+      +-------------------------------+---------------------------------------+
+      | (), [], ++, --                | postfix for call, index, incrementors |
+      +-------------------------------+---------------------------------------+
+      | +x, -x, ~x                    | unary operations                      |
```

Comment:
```suggestion
      | !, ~, -, **                   | Unary                      |
```

Reply:
Forgot `**`

Reply:
I intentionally take `**` from this PR because we need discussion.

In python, `**` is evaluated before unary. `-2 ** 2` is evaluated as `- (2 ** 2)` resulting `-4`. In addition, it is natural to add `**=` as a new assignment operator if `**` is introduced.

I will send a new PR that follows this to add `**`.

Reply:
Ah got it